### ISSUE TITLE
[Bug] column order changed to alphabetically 

### ISSUE THREAD

**Issue by dbartz** [yesoreyeram/yesoreyeram-boomtable-panel#151]
**Describe the bug**
Updating Grafana and the Plugin led to changes in the order of the columns:
* Before, the columns where sorted by the order of the series
* Now, they are ordered alphabetically

As a quick fix, I had to prepend column names with `0_` etc. - which is not that beautiful.

Imho the column order should be given by the _pattern_ order - or there should be an option to choose between series/pattern/alphabetical order.

**Version Details:**
 - Grafana version : 7.1.5
 - Plugin version : 1.4.0

**Comment by yesoreyeram** [comment]
Is the behaviour due to grafana upgrade or plugin upgrade?  Btw, what data source you are using ?

**Comment by dbartz** [comment]
Hi @yesoreyeram,
I frankly did not know, as we did not have fixed plugin versions.

In order to test I downgraded to plugin version 1.3.0, staying on Grafana 7.1.5:
This reverted from alphabetical column order to series order.
We are using graphite as a data source.

What is the desired behaviour?

**Comment by mykllterry** [comment]
I'll chime in with a "me too" 
I would prefer the order method to be selectable/

Version Details:

Grafana version : 7.2.1
Plugin version : 1.4.0

**Comment by limptable** [comment]
I would also prefer if the order was selectable in some way. Alphabetical is messing with our preferred sorting order. 

Version Details:
Grafana: 7.2.0
Plugin: 1.4.0

**Comment by fyang13** [comment]
Yes, it's very often user want the column to be ordered based on its own use case, not alphabetically. The behavior seems changed from https://github.com/yesoreyeram/yesoreyeram-boomtable-panel/pull/97 ?
Before that, user can change their query in Grafana to make the column display as they prefer, but now the columns are forced to be display alphabetically take away the choice from user.
One problem here is now we can't even use the old plugin to avoid this since Grafana v8 require signed plugin and old plugin version are unsigned. :(

**Comment by fyang13** [comment]
@yesoreyeram Note that for the column name using "font awesome icons", which has prefix with "_fa-", the force sorting on alphabetical order become completely useless, since it will put "_fa-clock-o_ Z" in front of "_fa-external-link_ A". So [this](https://github.com/yesoreyeram/yesoreyeram-boomtable-panel/pull/97) is really a broken change and probably should be reverted?
